backend/.env.example

```
@@ -0,0 +1,66 @@
+# Stripe Payment Configuration
+STRIPE_SECRET_KEY=sk_test_your_stripe_secret_key
+STRIPE_PUBLISHABLE_KEY=pk_test_your_stripe_publishable_key
+STRIPE_WEBHOOK_SECRET=whsec_your_webhook_secret
+
+# Email Configuration (SendGrid/Mailgun/SMTP)
+EMAIL_SERVICE=smtp
+EMAIL_HOST=smtp.gmail.com
+EMAIL_PORT=587
+EMAIL_USER=your_email@gmail.com
+EMAIL_PASSWORD=your_email_password
+EMAIL_FROM=Legion Command Center <noreply@legioncommand.com>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMAIL_FROM key should go before the EMAIL_HOST key

(UnorderedKey)